apis/infrastructure/v1beta1/byohost_webhook_internal_test.go

```diff
@@ -0,0 +1,198 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
```

Comment:
Are we following any naming conventions for the integration tests and unit tests files? Unit tests should be by default byohost_webhook_test.go, and if we are adding any other tests like integration tests, it should go under byohost_webhook_integration_test.go, but that's a nit picking. Just wanted to make sure we are consistent with the naming.

Reply:
The convention we are following is to test the public APIs(types, methods) i.e. BlackBox testing and hence both the unit and integration tests can go in package_test.go file. If in case we have to do WhiteBox testing i.e. to test the unexported APIs, we are creating test files as package_internal_test.go